lambdas/call-ticf-cri/src/test/java/uk/gov/di/ipv/core/callticfcri/CallTicfCriHandlerTest.java

```diff
@@ -294,7 +294,7 @@ void handleRequestShouldReturnJourneyErrorResponseIfCiStoringServiceThrows() thr
                 List.of(
                         new SqsException("Oops"),
                         new ParseException("Oops", 1),
-                        new JsonProcessingException("") {},
+                        new AuditExtensionException("") {},
```

Comment:
I think you can drop the `{}` from this line. I don't think it should have been there in the first place. This is creating [an anonymous class](https://docs.oracle.com/javase/tutorial/java/javaOO/anonymousclasses.html), which we don't really need.

Reply:
Reviewed, I wasnt sure what it was doing there in the first place and left it just in case